src/utils/filter.js

```diff
@@ -5,6 +5,6 @@ import {isEventPast, isEventFuture} from "./event";
 
 export const FILTERS = {
   [FilterType.EVERYTHING]: (events) => events,
```

Comment:
Сразу не заметил, но надо подправить. Здесь получается неконсистентно, т.к. в остальных случаях возвращается новый массив, а в этом тот же. Можно случайно его изменить, что может привести к ошибке.
```suggestion
  [FilterType.EVERYTHING]: (events) => [...events],
```